<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReverbDialog</class>
 <widget class="QDialog" name="ReverbDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>183</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>เอฟเฟ็กต์เสียงก้อง : Reverb</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="SwitchButton" name="btnSwitch" native="true">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>21</height>
        </size>
       </property>
       <property name="toolTip">
        <string>เปิด / ปิด  เสียงก้อง</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnReset">
       <property name="text">
        <string>รีเซ็ต</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>24</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QDial" name="dialInGain">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>51</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>51</height>
            </size>
           </property>
           <property name="minimum">
            <number>-96</number>
           </property>
           <property name="maximum">
            <number>0</number>
           </property>
           <property name="notchTarget">
            <double>2.700000000000000</double>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>InGain</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinInGain">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="minimum">
              <number>-96</number>
             </property>
             <property name="maximum">
              <number>0</number>
             </property>
             <property name="value">
              <number>-96</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QDial" name="dialRvTime">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>51</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>51</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>3000</number>
           </property>
           <property name="notchTarget">
            <double>34.700000000000003</double>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>ReverbTime</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinRvTime">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>3000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QDial" name="dialRvMix">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>51</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>51</height>
            </size>
           </property>
           <property name="minimum">
            <number>-96</number>
           </property>
           <property name="maximum">
            <number>0</number>
           </property>
           <property name="notchTarget">
            <double>2.700000000000000</double>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>ReverbMix</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinRvMix">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="minimum">
              <number>-96</number>
             </property>
             <property name="maximum">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_4">
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QDial" name="dialHF">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>51</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>51</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="notchTarget">
            <double>34.700000000000003</double>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>HighFreqRTRatio</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="spinHF">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>0.001000000000000</double>
             </property>
             <property name="maximum">
              <double>0.999000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header>SwitchButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
